MatrixKit/Models/Room/MXKRoomDataSource.m

```diff
@@ -1644,16 +1690,14 @@ - (void)processQueuedEvents:(void (^)())onComplete
                         [[NSNotificationCenter defaultCenter] postNotificationName:kMXKRoomDataSourceSyncStatusChanged object:self userInfo:nil];
                     }
                 }
+            
+                [self refreshUnreadCounters:NO];
```

Comment:
You should check whether self has not been destroyed before refreshing unread counters (see the following block)